addie/calculate_gr/edit_sq_dialog.py

```diff
@@ -243,9 +243,10 @@ def do_set_scale_range(self):
             current_scale_factor = max_scale
 
         # TODO/ISSUE/NOW - clean up to multiple steps and check!
-        scale_factor_int = int(current_scale_factor/(max_scale - \
-                                                     min_scale)*(self.ui.horizontalSlider_scale.maximum() -
-                                                                 self.ui.horizontalSlider_scale.minimum()))
+        delta_scale = max_scale - min_scale
+        delta_slider_scale = self.ui.horizontalSlider_scale.maximum() - self.ui.horizontalSlider_scale.minimum()
+        scale_factor_int = int(current_scale_factor/(delta_scale - delta_slider_scale))
```

Comment:
This is a different equation than the original one

Reply:
oups!!

Reply:
and oups again, fixed this time 
99b8738d496b6de4b86463fea4b71263d66a9d38